state/reducers/rootReducer.js

```diff
@@ -1,11 +1,17 @@
+import { action } from "commander";
```

Comment:
Why are you importing this here?

Reply:
honestly... no ide :O. but I have removed it